### ISSUE TITLE
Production Wars

### ISSUE THREAD

**Issue by ever-Lord** [explosivegaming/factorio-olympics#46]
**Feature overview**
Name: Randomize Map and Objectives

**Details**
- Seed should be random each new game (but still the same for all teams)
- there could be 2 to 4 Objectives, ie 2 to 4 different ingredients/products to build (same for all teams)
- each ingredient/product has a randomized quantity requested (same for all teams), the point is to have to decide between 2 strategies : manual craft vs automatisation (or anything between)
- the range between min and max quantity needs a factor of 100 (or may be even more)

**Comment by Cooldude2606** [comment]
Some findings:
- The map used is random, the issue is there are only two map configs, this should be expanded.
- The game is currently set to a maximum of 3 as that is what the base game had.
- The quanitiy requested is already randomized.
- Min and max have different ranges based on how many tasks are selected. For one task the range is 2x to 10x the value in the config, example: given batteries you can be asked to produce between 800 and 4,000 accumators, it would not be fun to change this to 800 and 80,000 it would just take too long.

TL;DR There should be more maps to select from, and to see if changing the max tasks to 4 will break anything.
